src/components/select-menu/_select-menu.scss

```diff
@@ -0,0 +1,329 @@
+@if ($includeHtml) {
+  .sg-select-menu {
+    --select-background: #{$gray-20};
```

Comment:
we should start using css variables as a values for tokens instead of scss variables. This shift will be needed in the future to support dark mode.

cc @clxandstuff @katarzynatobis